src/main/resources/defaults/bedrock-titan-embedding-defaults.json

```diff
@@ -0,0 +1,17 @@
+{
+    "template.name": "deploy-bedrock-titan-embedding-model",
+    "template.description": "Deploying Amazon Bedrock Titan embedding model ",
+    "create_connector.name": "Amazon Bedrock Connector: embedding",
+    "create_connector.description": "The connector to bedrock Titan embedding model",
+    "create_connector.region": "us-east-1",
+    "create_connector.endpoint": "api.openai.com",
+    "create_connector.credential.access_key": "123",
```

Comment:
Should we be including dummy credential values in these defaults?

Reply:
its actually just there cause its either going to be the "123" here or `${{create_connector.credential.key}}`

I could potentially fail for all templates with create_connector in them if no key is passed. I was actually thinking of doing this but didnt get to it. 

I would also do something similar for like semantic search template (the one without model creation), I could fail if no model ID is provided